code-challenges.test.js

```diff
@@ -54,3 +141,39 @@ const numbersToAdd3 = []
 
 
 // b) Create the function that makes the test pass.
+
+// Psuedocode:
+// 1. Create a function called accumulatingSum
+// 2. The function takes a parameter called array
+// 3. Create a variable called sum and set it to zero. This will be used to keep track of the accumulating total for each index.
+// 4. Create an array called result and this will be used as the return value
+// 5. Create a for loop to iterate through each index in the input array from the parameter
+    // 5a. Set the result index of i to equal to the sum and the array of the current index
+    // 5b. Set the sum to equal the result of the current i
+// 6. Exit the for loop and return the result that contains the array
+
+const accumulatingSum = (array) => {
+    let sum = 0
+    let result = []
+
+    for(let i=0; i<array.length; i++) {
+        result[i] = sum + array[i]
+        sum = result[i]
+    }
+    return result
+}
+
```

Comment:
Excellent work in this section!  Pseudocode is really thorough and clearly worded, and logic looks great for all problems.